go.mk

```diff
@@ -5,10 +5,9 @@ GITCOMMITTIMESTAMPUNIX ?= $(shell git log -1 --pretty=%ct)
 GITREFFILE ?= $(GITDIR)/$(shell git rev-parse --symbolic-full-name HEAD)
 GITROOT ?= $(shell git rev-parse --show-toplevel)
 
-# Ensure the Go version in .go-version is installed and used.
-GOROOT?=$(shell $(GITROOT)/script/run_with_go_ver go env GOROOT)
+GOROOT?=$(go env GOROOT)
 GOLANG_VERSION=$(shell cat $(GITROOT)/.go-version)
-GO:=$(GOROOT)/bin/go
+GO:=go
 GOARCH:=$(shell $(GO) env GOARCH)
 export PATH:=$(GOROOT)/bin:$(PATH)
```

Comment:
I think most of this is unnecessary now? PATH must already contain the go binary, and not sure if we need GOROOT otherwise.